docs/tutorial/population_dynamics.rst

```diff
@@ -278,6 +278,43 @@ These information criteria values weigh the log likelihood against the model com
 model selection. In general, the model with the lowest value is optimal. We can see that both values are lower for the double
 exponential model, but only slightly so it is not strong evidence to choose the more complex model.
 
+.. _pop_std_err:
+
+Standard errors
+^^^^^^^^^^^^^^^
+
+Obtaining lifetime estimates is important, but it is equally important to know how reliable these estimates are.
+One way to gauge the reliability of the estimates is to calculate the asymptotic standard errors of the parameters.
+We can find these in the `err_amplitudes` and `err_lifetimes` properties.
+
+    >>> dwell_1.err_lifetimes
+    array([0.00039664])
+
+    >>> dwell_2.err_amplitudes
+    array([0.03601527, 0.03601527])
+
+    >>> dwell_2.err_lifetimes
+    array([0.00072262, 0.00063035])
+
+When we have sufficient data, these approximate standard errors may be sufficient, but it should be noted that they are only asymptotically correct (for large amounts of data).
```

Comment:
I propose to mention the above code block, otherwise people will not read it.

Reply:
Something like `When we have sufficient data, the approximate standard errors calculated in the code block above may be sufficient, but it should be noted that they are only asymptotically correct (for large amounts of data).`?

Reply:
Sorry, I mistyped by comment. What I wanted to say, is that you should mention this above the code block. Otherwise people will not read it. In the above comment I already included the assumption about sufficient data, let me know what you think.

Reply:
Ohhh, but then it is indeed already covered by your previous suggestion. I will omit this sentence now then.